<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormatDBSupportRunDialog</class>
 <widget class="QDialog" name="FormatDBSupportRunDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>455</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>523</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Format database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="inputData">
     <property name="title">
      <string>Input data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="inputFilesRadioButton">
        <property name="text">
         <string>Select input file(s) for formatting database:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="inputFilesLayout">
        <item>
         <widget class="QLineEdit" name="inputFilesLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="inputFilesToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="inputDirRadioButton">
        <property name="text">
         <string>Or select directory with input files:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="inputFilesLayout_2">
        <item>
         <widget class="QLineEdit" name="inputDirLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="inputDirToolButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="filterGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>File filter</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QRadioButton" name="includeFilterRadioButton">
             <property name="text">
              <string>Include files filter</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="includeFFLineEdit">
             <property name="text">
              <string>*.fa;*.fasta</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="excludeFilterRadioButton">
             <property name="text">
              <string>Exclude files filter</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="excludeFFLineEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>*.pal</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Type of file(s)</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="proteinTypeRadioButton">
           <property name="text">
            <string>protein</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="nucleotideTypeRadioButton">
           <property name="text">
            <string>nucleotide</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputData">
     <property name="title">
      <string>Output settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Select the path to save the database into:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="databasePathLayout">
        <item>
         <widget class="QLineEdit" name="databasePathLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="databasePathToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Base name for BLAST files:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="baseNamelineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Title for database file:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="databaseTitleLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="formatButton">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inputDirRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>inputDirLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputDirRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>inputDirToolButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputFilesRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>inputFilesLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputFilesRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>inputFilesToolButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputDirRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>includeFilterRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>includeFFLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excludeFilterRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>excludeFFLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
